### ISSUE TITLE
Rawtherapee OSx 13.2, 13.3, 13.3.1

### ISSUE THREAD

**Issue by marcoandresdz** [Beep6581/RawTherapee#6743]
As soon as upgrading to OSX Ventura 13.2, Rawtherapee crashed shortly after launching. First observed when trying to process a dng file in Gimp. Rawtherapee crashed when trying to load the file. But Rawtherapee would launch.

After upgrading to 13.3, Rawtherapee won't even launch, so I cannot submit traceback/debugging information

The heart of the message:
**Termination Reason:    Namespace DYLD, Code 1 Library missing
Library not loaded: /Applications/RawTherapee.app/Contents/Frameworks/libomp.dylib**

[crash_report.pd
[crash_13_3_1.pdf](https://github.com/Beep6581/RawTherapee/files/11183540/crash_13_3_1.pdf)
[crash_13_3t.pdf](https://github.com/Beep6581/RawTherapee/files/11183541/crash_13_3t.pdf)
f](https://github.com/Beep6581/RawTherapee/files/11183489/crash_report.pdf)
Resolution steps [advised after contacting Apple]: 

- Delete and reinstall
- Run in Recovery mode [reinstalled system]
- Run in safe mode

13,2, 13,3, and 13.3.3 Since I managed to reproduce the crash problem in safe mode, it is not an extension problem. Deleting and reinstalling did solve the problem either.

Basically it has not worked since before 13.2 was installed.

[crash_13_3_1.pdf](https://github.com/Beep6581/RawTherapee/files/11183640/crash_13_3_1.pdf)

**Comment by Lawrence37** [comment]
Did you install it directly into the /Applications folder?

**Comment by marcoandresdz** [comment]
Lawrence,
Yes, I dragged the application to the applications folder after unzipping.
Andrés

After installing Ventura 13.3.1, Rawtherapee launches but then immediately crashes. 
￼

Can’t look at the log because the system is trying to send the report to Apple and does not stop. Will need to reboot again.

￼

￼
￼

System crashed for another reason just now:
￼
Rebooted

**Comment by Lawrence37** [comment]
Your crash report shows the application path as `/Applications/rawtherapee/RawTherapee.app/Contents/MacOS/rawtherapee`. I'm not a macOS package expert, but I think `RawTherapee.app` should be directly in `/Applications` instead of `/Applications/rawtherapee`. @Benitoite can confirm.

Note: possible duplicate of #6580

**Comment by marcoandresdz** [comment]
Lawrence.
In the meantime  I’ll convert negs using Adobe Digital Negative Converter and Darktable. Prefer doing everything in Gimp with handoff to Rawtherapee. They must have had an issue with 13.2. They’ve released 13.3 and 13.3.1 after.

Thank you,
 Marco Andrés
Same message as before:
￼

**Comment by Lawrence37** [comment]
The images (?) in your messages are not appearing. Instead they show as ￼:. Maybe it is not possible to embed images through email replies.
> Same message as before:
> ￼

**Comment by marcoandresdz** [comment]
﻿LawrenceYour hunch was correct. After moving Rawtherapee into the top level of the Applications folder, it worked and could read DNG files. Even after setting preferences in Gimp to open raw/dng files in Rawtherapee and Darktable, Gimp cannot read the processed files. While Gimp passes control to the application when « reading » the DNG, upon return, there is an errorThe problems began after upgrading the OS. First a boot problem (kept rebooting). Then Rawtherapee and Darktable could not read the dng Silverfast produced, although they could before the upgrade. Used Adobe DNG Converter to fix the dng files. While Darktable was able to read the file, Rawtherapee could not since it was not in the top-level of the Applications folder.I’ll investigate more before filing a bug report.Let’s close the bug.Thank you,AndrewOn 8 Apr 2023, at 22:00, Lawrence37 ***@***.***> wrote:﻿
The images (?) in your messages are not appearing. Instead they show as ￼:. Maybe it is not possible to embed images through email replies.

Same message as before:
￼

**Comment by marcoandresdz** [comment]
Lawrence,
The install of the latest version of Ventura 13.3.1 has corrected the problem with Silverfast creating DNG files without the sidecar with extension “pp3”. No longer need to use Adobe DNG Converter.

I’ve attached a PDF with the image, wince you were unable to see them before. Will report problem resolution to Silverfast. Now the problem appears to be with GIMP, since Rawtherapee and Darktable don’t play well with Gimp now but did before.

Thank you,
Marco Andrés
￼

**Comment by Lawrence37** [comment]
Glad you were able to resolve the RawTherapee error.

The PDF did not upload. In the future, you'll need to upload them directly on GitHub instead of replying via email.

Closing as a duplicate of #6580.
